general/clients/codecs.md

```diff
@@ -0,0 +1,89 @@
+|:---:|:---:|:---:|:---:|:---:|:---:|
+|H.262|MPEG-2 Video|MPEG4 SP/ASP|H.264|MP4 Container<sup>2</sup>|H.265|
+|DVD-Video||DivX|MPEG-4 AVC||HEVC|
+|||DX50||||
+
+<sup>1</sup>[MPEG-4 Part-2 vs Part-10](https://www.afterdawn.com/glossary/term.cfm/mpeg_4_part_10)
+
+<sup>2</sup>[MPEG-4 Part 17: MP4TT Subtitles](https://en.wikipedia.org/wiki/MPEG-4_Part_17)
+
+## [Audio Compatibility](https://en.wikipedia.org/wiki/Comparison_of_video_container_formats#Audio_coding_formats_support "Wikipedia's audio codec tables")
+
+If the audio codec is unsupported or incompatible (such as playing a 5.1 channel stream on a stereo device), the audio codec must be transcoded. This is not nearly as intensive as video coding.
+
+||Chrome|Firefox|Safari|Android|AndroidTV|Kodi|Roku|
+|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+FLAC|✅|❌|✅|✅|✅|✅|✅|
+|MP3|🔶<sup>1</sup>|🔶|✅|✅|✅|✅|✅|
+|AAC|🔶<sup>2</sup>|🔶|✅|✅|✅|✅|✅|
+|AC3|✅|❌|✅|✅|✅|✅|✅|
+|EAC3<sup>3</sup>|✅|✅|✅|✅|✅|✅|✅|
+|VORBIS|❌|✅|✅|✅|✅|✅|✅|
+|DTS<sup>4</sup>|❌|❌|❌|✅|✅|✅|✅|
+
+<sup>1</sup>MP3 Mono is incorrectly reported as unsupported and will transcode to AAC.
+
+<sup>2</sup> AAC is incorrectly reported as unsupported and will transcode to MP3.
+
+<sup>3</sup>Only EAC3 2.0 has been tested.
+
+<sup>4</sup> Only DTS Mono has been tested.
+
+## [Subtitle Compatibility](https://en.wikipedia.org/wiki/Comparison_of_video_container_formats#Subtitle/caption_formats_support "Wikipedia's subtitle codec tables")
+
+Subtiles can be a subtle issue for transcoding. Containers have a limited number of subtitles that are supported. If subtitles need to be transcoded, it will happen one of two ways. They can be converted into another supported format (text-based subtitles) or burned into the video (image/lossless based and ASS based) due to the subtitles transcoding not being supported. This is the most intenstive method of transcoding due to two transcodings happening at once; applying the subtitle layer on top of the video layer.
+
+||Format|TS|MP4|MKV|AVI|
+|:---:|:---:|:---:|:---:|:---:|:---:|
+|SubRip Text (.srt)|Text|❌|🔶|✅|🔶|
+|ASS/SSA<sup>1</sup>|Formatted Text|❌|❌|✅|🔶|
+|VobSub|Picture|❌|✅|✅|🔶|
+|DVB-SUB [(.sub/.idx)](https://forum.videohelp.com/threads/261451-Difference-between-SUB-and-IDX-file)|Picture|✅|❌|✅|❌|
+|MP4TT/TXTT|XML|❌|✅|❌|❌|
+|PGSSUB|Picture|❌|❌|✅|❌|
+
+
+<sup>1</sup>ASS Subtitles are only supported by MKV files. MKV files aren't supported by Firefox.They will always inherently be burned into the video. This is not a limitation of Jellyfin.
+
+## [Container Compatibility](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Containers)
+
+If the container is unsupported, this will result in remuxing. The video and audio codec will remain intact, but wrapped in a container that is supported. This is the least intensive process. Most video containers will be remuxed to use the hls streaming protocol and ts containers. Remuxing shouldn't be a concern even for a RPi3.
```

Comment:
```suggestion
If the container is unsupported, this will result in remuxing. The video and audio codec will remain intact, but wrapped in a supported container. This is the least intensive process. Most video containers will be remuxed to use the HLS streaming protocol and TS containers. Remuxing shouldn't be a concern even for an RPi3.
```